pkg/server/server.go

```diff
@@ -272,8 +272,17 @@ func stageFiles(ctx context.Context, sc *Context, controlConfig *config.Control)
 		return err
 	}
 	dataDir = filepath.Join(controlConfig.DataDir, "manifests")
+
+	// extraDNS prepares a slice with the DNS IPs to be passed as clusterIPs field in the kube-dns service
+	extraDNS := "["
+	for _, dns := range controlConfig.ClusterDNSs {
+		extraDNS = extraDNS + dns.String() + ","
+	}
+	extraDNS = extraDNS + "]"
+
 	templateVars := map[string]string{
 		"%{CLUSTER_DNS}%":                 controlConfig.ClusterDNS.String(),
+		"%{CLUSTER_EXTRA_DNS}%":           extraDNS,
```

Comment:
```suggestion
		"%{CLUSTER_DNS_LIST}%":           fmt.Sprintf("[%s]", util.JoinIPs(controlConfig.ClusterDNSs)),
```

Reply:
aaaaggghhh!! I knew that function but I forgot about it! Thanks Brad, great idea